### PR TITLE
cmd/evm: Allow stdin and files as sources of bytecode

### PR DESCRIPTION
**Pull request by Arachnid**

### CODE REVIEW

File: cmd/evm/main.go [hunk 1]
```diff
@@ -133,12 +139,33 @@ func run(ctx *cli.Context) error {
...
 
+	if ctx.GlobalString(CodeFlag.Name) != "" {
+		code = common.Hex2Bytes(ctx.GlobalString(CodeFlag.Name))
+	} else {
+		var hexcode []byte
+		if ctx.GlobalString(CodeFileFlag.Name) != "" {
+			var err error
+			hexcode, err = ioutil.ReadFile(ctx.GlobalString(CodeFileFlag.Name))
+			if err != nil {
+				panic(err)
```
Comment (obscuren) [inline cmd/evm/main.go]: Can we not `panic` here? Can't we just print and exit?

Comment (Arachnid) [inline cmd/evm/main.go]: PTAL.

**Comment by fjl** [comment]
@Arachnid please fix `os.exit` -> `os.Exit`.

**Comment by karalabe** [comment]
@fjl @Arachnid Fixed this PR so we can merge.

**Pull request merged**